appinfo/info.xml

```diff
@@ -17,7 +17,7 @@
 - 🚀 Get your project organized
 
 </description>
-    <version>1.4.0-alpha1</version>
```

Comment:
If you need to bump the version for migrations, please only bump the `alpha1` to `alpha2`, the regular version bump will be done during the release process.